src/store/greetingsSlice.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  greetings: [],
+  loading: false,
+};
+
+const greetingsSlice = createSlice({
+  name: 'greetings',
+  initialState,
+  reducers: {
+    setGreetings: (state, action) => ({
+      ...state,
+      greetings: action.payload,
+    }),
+  },
+});
+
+export const { setGreetings } = greetingsSlice.actions;
+
+export default greetingsSlice.reducer;
+
+const fetchGreetings = () => async (dispatch) => {
+  try {
+    const res = await fetch('//localhost:3000/api/v1/greetings');
+    const data = await res.json();
+    dispatch(setGreetings(data));
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
```

Comment:
- [ ] For **[Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)** and clean code reasones, kindly ensure that you **fix your linters errors instead of disabling them**, you should remove the `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.